miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/MiniAppWebView.kt

```diff
@@ -134,7 +134,7 @@ internal open class MiniAppWebView(
     override fun runErrorCallback(callbackId: String, errorMessage: String) {
         post {
             evaluateJavascript(
-                "MiniAppBridge.execErrorCallback(\"$callbackId\", \"$errorMessage\")"
+                "MiniAppBridge.execErrorCallback(`$callbackId`, `${errorMessage.replace("'", "\\'")}`)"
```

Comment:
Same here - maybe we should have a unit test for escaped characters?

Also, I just noticed that `runSuccessCallback` above is escaping backticks - I don't remember the particular reason that was needed, but maybe we should have a shared escaping function for both `runSuccessCallback` and `runErrorCallback` to ensure that quotation marks and backticks are escaped for both? What do you think?

Reply:
Oh, I just noticed: the response in `runSuccessCallback` is being sent with backticks surrounding it, so that's why it needs to escape backticks. So I think here maybe you need to do the same thing here - escape backticks instead of single quotes

Reply:
Updated with backtick escape and unit test.